app/controllers/admin/home_controller.rb

```diff
@@ -5,7 +5,45 @@ class HomeController < Admin::BaseController
     skip_after_action :verify_authorized, only: :show
     skip_after_action :verify_policy_scoped, only: :show
 
-    # this isn't used yet
-    def show; end
+    # show 2023 pilot stats
+    def show
+      @pilot_stats = get_pilot_stats
+    end
+
+  private
+
+    def get_pilot_stats
+      choices = programme_choices
+
+      OpenStruct.new({
+        cip_total: choices.fetch("core_induction_programme", 0),
+        diy_total: choices.fetch("design_our_own", 0),
+        fip_total: choices.fetch("full_induction_programme", 0),
+        no_ects_total: choices.fetch("no_early_career_teachers", 0),
+        total: choices.values.sum,
+        partnership_totals:,
+      })
+    end
+
+    def programme_choices
+      SchoolCohort.where(cohort:).group(:induction_programme_choice).count
+        .reject { |choice| choice.in? %w[school_funded_fip not_yet_known] }
+    end
+
+    def partnership_totals
+      totals = Partnership.unchallenged.where(cohort:).group(:lead_provider_id).count
+
+      providers.map do |provider|
+        { name: provider.name, total: totals.fetch(provider.id, 0) }
+      end
+    end
+
+    def providers
+      LeadProvider.where(id: ProviderRelationship.where(cohort:).select(:lead_provider_id)).order(:name)
+    end
```

Comment:
```suggestion
    def get_pilot_stats
      OpenStruct.new(
        cip_total: program_choices.fetch("core_induction_programme", 0),
        diy_total: program_choices.fetch("design_our_own", 0),
        fip_total: program_choices.fetch("full_induction_programme", 0),
        no_ects_total: program_choices.fetch("no_early_career_teachers", 0),
        total: program_choices.values.sum,
        partnership_totals:,
      )
    end
    
    def programme_choices
      @programme_choices ||= SchoolCohort.where(cohort:)
                                         .group(:induction_programme_choice)
                                         .count
                                         .except("school_funded_fip", "not_yet_known")
    end
    
    def partnership_totals
      Partnership
        .unchallenged
        .joins(:lead_provider)
        .where(cohort:)
        .order("lead_providers.name")
        .group("lead_providers.name")
        .count
    end
```
    def get_pilot_stats
      choices = programme_choices

      OpenStruct.new({
        cip_total: choices.fetch("core_induction_programme", 0),
        diy_total: choices.fetch("design_our_own", 0),
        fip_total: choices.fetch("full_induction_programme", 0),
        no_ects_total: choices.fetch("no_early_career_teachers", 0),
        total: choices.values.sum,
        partnership_totals:,
      })
    end

    def programme_choices
      SchoolCohort.where(cohort:).group(:induction_programme_choice).count
        .reject { |choice| choice.in? %w[school_funded_fip not_yet_known] }
    end

    def partnership_totals
      totals = Partnership.unchallenged.where(cohort:).group(:lead_provider_id).count

      providers.map do |provider|
        { name: provider.name, total: totals.fetch(provider.id, 0) }
      end
    end

    def providers
      LeadProvider.where(id: ProviderRelationship.where(cohort:).select(:lead_provider_id)).order(:name)
    end
```

Reply:
Do not commit if you don't like it.

Reply:
Thanks, not a massive fan of this in this case, I deliberately kept this simple and this is only a temporary page to enable visibility of the stats that I pull out each day.

Reply:
I'm not sure I get your comment @tonyheadford 
I think my solution is simpler than the existing one.

Reply:
Oh sorry @ltello I was rushing on Friday and I missed what you actually changed and thought you were just reformatting the whole file. By simple I meant that the code was very simple and, in my head at least, easy to understand, it's just lifted from the console queries I run. I didn't communicate that this was a quick and dirty temporary page when I asked for a sanity check, so my bad, but it's good that you picked up on I could improve the queries, thanks.

Reply:
Also my original version was designed to allow us to show which providers _haven't_ partnered yet which I think is missed in your version.

Reply:
Well, if you think it adds value to display the rest of lead providers with 0 count then we can leave everything as it was.

Reply:
Yes, definitely, I know Kate is interested in why there's one hasn't entered any and I think given in prod there are very few providers (unlike the loads in the dev env) it makes it clear to see all.  Plus it was in the mockup @frankieroberto built.

Reply:
Yep, I’d definitely like to see all providers listed, even if they have 0 schools signed up. 👍

Reply:
@tonyheadford @leoapost are we happy to resolve this comment?